src/app/frontend/deploy/deployfromfile.html

```diff
@@ -0,0 +1,35 @@
+<!--
+Copyright 2015 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<div layout="row" layout-align="space-between start">
+  <div layout="column" flex="80">
```

Comment:
It is redundant. I think just a child div element with flex attribute is enough to use the grid. I will remove layout attribute.